.changes/1.0.0.md

```diff
@@ -0,0 +1,34 @@
+## dbt-snowflake 1.0.1-rc1 - April 19, 2022
```

Comment:
I think this file isn't right.  It looks like you may have combined all the 1.0.0 changelogs with the 1.0.1 changelogs.  They should not be under the same heading.

Reply:
split out between 1.0.0 and 1.0.1 in files then ran new changie merge